setup.py

```diff
@@ -11,7 +11,8 @@
 
 
 setuptools.setup(
-    python_requires='>=3.6',
-    cmdclass=cmd_class,
-    setup_requires=['pbr'],
-    pbr=True)
+    packages=setuptools.find_packages(
+        where='.',
+        include=['testtools*'],
+    ),
+    cmdclass=cmd_class)
```

Comment:
Is there any reason to keep `setup.py` now? You already have a:
```
[tool.setuptools.packages.find]
include = ["testtools"]
```
section and you should be able to also do this in `pyproject.yaml`:

```
[tool.hatch.build]
include = ["/testtools"]

[tool.hatch.build.targets.sdist]
include = ["/testtools"]
```

(I've not tried this so you might need to tweak paths etc)

Reply:
We still need to register TestCommand. It's probably possible to do that via pyproject, but I'd rather just punt on that since we've said we'll drop TestCommand anyway at the next major release.